frontend/src/utils/FileUtils.ts

```diff
@@ -0,0 +1,11 @@
+// eslint-disable-next-line import/prefer-default-export
+export const downloadFile = (data: string, fileName: string): void => {
```

Comment:
Since the file endpoint and mutation returns a Firebase URL, I think this means we can skip the `const url = URL.createObjectURL(data);` step?

Reply:
yup 👍 haven't fully modified it yet. thanks for double checking.